### PR TITLE
Raise Change Request

### PR DESCRIPTION
**Pull request by andrewfraser73**
the ability to do simple change request creation
only currently allow creation of non-board assembly change requests

### CODE REVIEW

File: src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        collectionSelectorHelpers.getSearchLoading(state.parts)
+    );
+
+    const handlePartChange = selectedPart => {
+        handleFieldChange('newPartNumber', selectedPart.partNumber);
+    };
+
+    return (
+        <>
+            {' '}
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js:35]: whitespace - no sure if intentional or if the linter sneaked this in

File: src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    const create = () => {
+        dispatch(changeRequestActions.add(item));
+    };
+
+    const createdItem = useSelector(reduxState =>
+        itemSelectorHelpers.getItem(reduxState.changeRequest)
+    );
+
+    useEffect(() => {
+        if (createdItem?.documentNumber) {
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js:67]: is this effect just to redirect to the newly created thing? Doesn't the itemCreated middleware handle that for us usually?

**Review by lewisrenfrew — APPROVED** [review]

File: src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js (hunk 2 above)
Comment (andrewfraser73) [reply src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js:67]: it didn't do it for me so i put this in

Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js:67]: you might need a rel: 'self' link on your resource

Comment (andrewfraser73) [reply src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js:67]: yeah put the self link in and i was able to remove this code

**Pull request merged**